dashboard/app/assets/stylesheets/application.scss

```diff
@@ -3291,3 +3291,55 @@ h2.danger {
     color: $yellow;
   }
 }
+
+.foorm-reset-font {
+  font-family: inherit !important;
+}
+
+.foorm-adjust-body {
+  border: none !important;
```

Comment:
for my information: what does !important do?

Reply:
It's not an ideal solution, but it overrides the default application order of CSS: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity